### DISCUSSION TITLE
Help me. Output size is smaller than input size. Is that normal ?

### DISCUSSION THREAD

**Discussion by lizat1** [TheLastGimbus/GooglePhotosTakeoutHelper#185] [I NEED HELP] [answered]
I'm new to this tool. Just download it and works fine.
But I have one concern the photos/videos that I downloaded from Google Photos have a total of 91Gb. When I use GooglePhotosTakeoutHelper to help me extract and organize files the final output has only 72Gb. Is that normal?

**Comment by TheLastGimbus** [comment, marked as answer]
This seems very fine

You see, the way takeout handles albums is it just copies the photos. This means they are plain duplicates of photos that are already in "Photos from 20..."

gpth doesn't yet support album organization because of this precise reason - i don't want to waste space with duplicates 0_o

so it's probable that free space came from **not duplicating photos**

TL;DR Don't worry, they are all there, just not album-ed

Ps. It's nice you noticed and reported this - nice to have aware users 🧠🧠

**Reply by lizat1** [reply]
Oh, thank you for the answer. I got it now. I tried to unzip it manually and exactly like you said, there are many duplicated photos from other albums that were already put in the Photos from 20XX. Nice feature to remove them.

**Reply by TheLastGimbus** [reply]
Psst, albums got implemented if you want them :see_no_evil: #187
